data/region-overrides.json

```diff
@@ -807,15 +807,6 @@
       "region": "36061",
       "context": "NY County, NY big assumed backlog on 5/9/2022"
     },
-    {
-      "include": "region",
-      "metric": "metrics.caseDensity",
-      "blocked": true,
-      "start_date": "2022-05-10",
-      "end_date": "2022-05-10",
-      "region": "PA",
-      "context": "PA staet-level cases spike on 5/11/2022"
```

Comment:
Oh, haha. I guess we intentionally removed this as an outlier.  I somehow missed that and assumed it was outlier detection. 🤦

Reply:
oops 😭 looks like I added that but I have no recollection of it--sorry for not realizing that hahah 🤦

Reply:
Heh. Nah, it's 100% my bad for not noticing it when I was diagnosing the issue of low cases. All good.